src/ssl.c

```diff
@@ -57761,80 +57761,77 @@ static const conf_cmd_tbl conf_cmds_tbl[] = {
 static const size_t size_of_cmd_tbls = sizeof(conf_cmds_tbl)
                                                     / sizeof(conf_cmd_tbl);
 
-/**
- * send configuration command
- * @param cctx  a pointer to WOLFSSL_CONF_CTX structure
- * @param cmd   configuration command
- * @param value arguments for cmd
- * @return 1 when cmd is recognised, but value is not used
- *         2 both cmd and value are used
- *  otherwise WOLFSSL_FAILURE
- *         -2 if cmd is not recognised
- *         -3 if value is NULL, but cmd is recognized
- */
-int wolfSSL_CONF_cmd(WOLFSSL_CONF_CTX* cctx, const char* cmd, const char* value)
+static const conf_cmd_tbl* CONF_find_cmd(WOLFSSL_CONF_CTX* cctx,
```

Comment:
This API looks like the OpenSSL one now. Rename to wolfssl_conf_find_cmd() to make it clear.

Reply:
Renamed.

Reply:
Sorry, but the name hasn't changed in the pull request. Did you push up the change?

Reply:
D'oh! Forgot to push the changes. Pushed now so you should see the rename.